release_notes/Service_Fabric_BreakingChanges.md

```diff
@@ -0,0 +1,110 @@
+
+
+## Breaking Changes 82CU1
+
+| Versions | Area | Description | 
+|-|-|-|
+| **Windows - 8.2.1363.9590<br>Ubuntu 18 - 8.2.1204.1** | Windows Server EOL | Support for Windows Server 2016 and Windows Server 1809 will be discontinued in future Service Fabric releases. We recommend updating your cluster VMs to Windows Server 2019.
+
+## Breaking Changes 82
+
+| Versions | Area | Description | 
+|-|-|-|
+| **Windows - 8.2.1235.9590<br>Ubuntu 18 - 8.2.1124.1** | ASP.NET Core 1.0 Support | Service Fabric ASP.NET Core packages are built against ASP.NET Core 1.0 binaries which are out of support. Starting Service Fabric 8.2, we will be building Service Fabric ASP.NET Core packages against .NET Core 3.1, .NET Core 5.0, .NET Core 6.0, .NET Framework 4.6.1. As a result, they can be used only to build services targeting .NET Core 3.1, .NET Core 5.0, .NET Core 6.0, >=.NET Framework 4.6.1 respectively.<br> For .NET Core 3.1, .NET Core 5.0, .NET Core 6.0, Service Fabric ASP.NET Core will be taking dependency on Microsoft.AspNetCore.App shared framework, whereas for NetFx target frameworks >=.NET Framework 4.6.1, Service Fabric ASP.NET Core will be taking dependency on ASP.NET Core 2.1 packages.<br>The package Microsoft.ServiceFabric.AspNetCore.WebListener will no longer be shipped, Microsoft.ServiceFabric.AspNetCore.HttpSys package should be used instead 
+
```

Comment:
Missing 8.0 Upcoming b/c -- 

"Support for Windows Server 2016 Version 1809 will be discontinued in future Service Fabric releases. We recommend updating your cluster VMs to Windows Server 2019."

"Support for Windows 10 Version 1809 will be discontinued in future Service Fabric releases. We recommend updating your cluster VMs to a supported version of Windows 10."

Reply:
8.1 CU1 has the following upcoming b/c -- seems familiar -- might be a repeat -- 

".NET Core runtime LTS 2.1 runtime will go out of support from Aug 21, 2021. Service Fabric releases after that date will drop support for Service Fabric apps running with .NET Core 2.1 runtime. This includes the following release versions and beyond - 8.0 CU4 and above, 8.1 CU2 and above and 8.2 RTO release and above. Current apps running on .NET Core 2.1 runtime will continue to work, but requests for investigations or request for changes will no longer be entertained. Service Fabric .NET SDK will take a dependency on .NET runtime 3.* features to support Service Fabric .NET Core apps. This has no impact on Service Fabric .NET Framework SDK."

Reply:
missing 8.1 CU2 b/c -- 

"Ubuntu 16.04 LTS reached it's 5-year end-of-life window on April 30, 2021 and is no longer supported by it's vendors. Service Fabric runtime will drop support for Ubuntu 16.04 after that date as well. Current applications running on it will continue to work, but requests for investigations or requests for change will no longer receive support. We recommend moving your applications to Ubuntu 18.04."

Reply:
missing b/c -- 8.2 cu1 -- 

"Windows Server 2022: Service Fabric runtime has identified a bug during additional testing and is temporarily suspending support for Windows Server 2022. We are working to resume the support soon."